dev_course/dl2/02b_initializing.ipynb

```diff
@@ -6,7 +6,8 @@
    "metadata": {},
```

Comment:
<p>This diff needs to be reverted.</p>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai_docs/pull/59/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>